src/bindings/js/node/lib/addon.ts

```diff
@@ -12,10 +12,19 @@ type SupportedTypedArray =
   | Float64Array;
 
 interface Core {
-  compileModel(model: Model, device: string, config?: {[option: string]: string}): CompiledModel;
-  compileModelSync(model: Model, device: string, config?: {[option: string]: string}): CompiledModel;
+  compileModel(
+    model: Model,
+    device: string,
+    config?: { [option: string]: string }
+  ): Promise<CompiledModel>;
+  compileModelSync(
+    model: Model,
+    device: string,
+    config?: { [option: string]: string }
+  ): CompiledModel;
   readModel(modelPath: string, binPath?: string): Promise<Model>;
   readModelSync(modelPath: string, binPath?: string): Model;
+  readModelFromBuffer(modelBuffer: Buffer, binBuffer?: Buffer): Model;
```

Comment:
In read_from_buffer sample there is a possibility to pass three args

Reply:
Sample was leaved by mistake